src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Fragment/FragmentAlt.vue

```diff
@@ -3,22 +3,27 @@
     <div class="segment">
       <comment v-if="comment" :comment="comment" :commentObj="commentObj" />
 
-      <div
-        class="header bg-skin-fragment-header text-skin-fragment-header text-base leading-4 rounded-t"
-      >
-        <div class="name font-semibold p-1 border-b"><label class="p-0">Alt</label></div>
-      </div>
-      <div class="segment">
-        <div class="text-skin-fragment">
-          <label class="condition p-1">[{{ condition }}]</label>
+      <div class="header bg-skin-fragment-header text-skin-fragment-header text-base leading-4 rounded-t" >
+        <div class="name font-semibold p-1 border-b">
+          <div style="display: flex; width: 100%; justify-content: space-between;" >
+            <label class="p-0 mb-0">Alt</label>
+            <collapsible-button :element-id="`collapse-${uuid}`" />
+          </div>
         </div>
-        <block
-          v-if="blockInIfBlock"
-          :style="{ paddingLeft: `${offsetX}px` }"
-          :context="blockInIfBlock"
-          :selfCallIndent="selfCallIndent"
-        ></block>
       </div>
+      
+    </div>
+    <b-collapse visible :id="`collapse-${uuid}`" class="mt-2">
```

Comment:
why "mt-2"? It causes unnecessary margin.

Reply:
Removed.